sources/editor/Stride.GameStudio/Program.cs

```diff
@@ -234,6 +234,7 @@ private static async void Startup(UFile initialSessionPath)
                     var message = "Could not find a compatible version of MSBuild.\r\n\r\n" +
                                   "Check that you have a valid installation with the required workloads, or go to [www.visualstudio.com/downloads](https://www.visualstudio.com/downloads) to install a new one.\r\n" +
                                   "Also make sure you have the latest [.NET 6 SDK](https://dotnet.microsoft.com/) \r\n\r\n" +
+                                  "Detected SDK: " + RuntimeInformation.FrameworkDescription.ToString() + "\r\n\r\n" +
```

Comment:
This is still the runtime version though, not the sdk, right ? And I think it might be the runtime that the application is under, not the one installed. So if the gamestudio is self contained, which might be the case (?), this'll always report net6 ?

Reply:
Ah you're right, good catch! I've removed that addition, the change in FindAndSetMSBuildVersion() still works as intended. You can test this by uninstalling the .NET 6 SDK and running a previously built version of this PR.

Reply:
> gamestudio is self contained

I'm not sure that is the case. Looking at the contents of `Stride.GameStudio` NuGet package it is built as runtime dependent.

Reply:
> > gamestudio is self contained
> 
> I'm not sure that is the case. Looking at the contents of `Stride.GameStudio` NuGet package it is built as runtime dependent.

Hmm, when I tested before the final PR update the "Detected SDK" was still reporting .NET 6.0 despite the SDK/runtime being uninstalled. MSBuildLocator on the other hand did pick up that the SDK was missing.